ci/lint.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.15.7
```

Comment:
Use latest version of go for library
```suggestion
    tag: latest
```

Reply:
ok cool - this might need updating in other repos as well

Reply:
This only applies to libraries